app/src/main/java/com/sjh/sunflower_sksowk156/SunflowerApplication.kt

```diff
@@ -0,0 +1,34 @@
+package com.sjh.sunflower_sksowk156
+
+import android.app.Application
+import android.content.Context
+import android.util.Log
+import fi.iki.elonen.NanoHTTPD
+
+class SunflowerApplication : Application() {
+    var server: TempServer? = null
+
+    override fun onCreate() {
+        super.onCreate()
+        // 서버 시작
+        server = TempServer(this)
+        server?.start()
+    }
+
+    class TempServer(private val context: Context) : NanoHTTPD(8081) {
+        override fun serve(session: IHTTPSession?): Response {
+            return try {
+                val response =
+                    context.assets.open("plants.json").bufferedReader().use { it.readText() }
```

Comment:
```suggestion
                    context.assets.open("plants.json").bufferedReader().use(Reader.readText)
```
가 되던가요? 가능하면 람다 보다는 펑션 레퍼런스를 사용하는 편이
코드를 읽는 입장에서 타입이 눈에 보이기 때문에 예상하기가 용이합니다.

람다의 사용은 피하면 피할 수록 좋습니다.